lib/application.ts

```diff
@@ -179,6 +181,9 @@ export class SNApplication {
         return undefined;
       });
     this.createdNewDatabase = databaseResult?.isNewDatabase || false;
+    if (!this.fixedNamespace) {
+      await this.namespaceService!.initialize();
```

Comment:
`initialize` seems poorly named. It seems like it should be something like `assignNamespace`? Also, doesn't `deviceInterface!.openDatabase()` depend on the correct namespace being set? If so, seems like namespace initialization should be the first thing in prepareForLaunch.

Reply:
>initialize seems poorly named. It seems like it should be something like assignNamespace?

I was keeping the same standard, most of the services have an `initialize` method. In this case, this method just decides what namespace to use and switchs to it.